integration/benchmark/contLoadGenerator.go

```diff
@@ -286,6 +288,11 @@ func (lg *ContLoadGenerator) Stop() {
 	lg.workerStatsTracker.StopPrinting()
 	lg.log.Debug().Msg("stopping follower")
 	lg.follower.Stop()
+	lg.stoppedChannel <- true
+}
+
+func (lg *ContLoadGenerator) GetStoppedChannel() chan bool {
```

Comment:
you might consider to make the signature compatible with the `context`'s Done, e.g.: 
```Done() <-chan struct{}```